search/includes/classes/class-search.php

```diff
@@ -615,4 +629,15 @@ public function filter__epwr_score_mode( $score_mode, $formatted_args, $args ) {
 	public function filter__epwr_boost_mode( $boost_mode, $formatted_args, $args ) {
 		return 'multiply';
 	}
+
+	/*
+	 * Increment the number of queries that have been passed through VIP Search
+	 */
+	private static function query_count_incr() {
+		if ( ! wp_cache_get( self::$query_count_cache_key ) ) {
```

Comment:
Explicit not-exists check:

```suggestion
		if ( false === wp_cache_get( self::$query_count_cache_key ) ) {
```

Otherwise, if for some reason the value is already `0` we end up calling an unnecessary `set`.